shell/common/rasterizer.cc

```diff
@@ -135,8 +162,13 @@ void Rasterizer::DoDraw(std::unique_ptr<flutter::LayerTree> layer_tree) {
   PersistentCache* persistent_cache = PersistentCache::GetCacheForProcess();
   persistent_cache->ResetStoredNewShaders();
 
-  if (DrawToSurface(*layer_tree) == RasterStatus::kSuccess) {
+  RasterStatus raster_status = DrawToSurface(*layer_tree);
```

Comment:
Are we calling DrawToSurface twice on purpose?

Reply:
:man_facepalming: typo.